docs/source/thanos.rst

```diff
@@ -0,0 +1,79 @@
+Using Thanos With Scylla Monitoring
+============================
+
+Scylla-Monitoring uses `Prometheus <https://prometheus.io/>`_ for metrics collection, which works out-of-the-box, but Prometheus does have limitations.
+`Thanos <https://thanos.io/>`_  is an opensource solution on top of Prometheus with multiple functionalities it can:
+
+1. Support high-availability.
+2. Support horizontal scaling.
+3. Support backup.
+
+A nice thing with Thanos approach is its flexibility, you can use some of its functionality depends on your need.
+
```

Comment:
```suggestion
In the rest of this guide we will learn how to deploy a Thanos cluster as a HA back-end for Grafana in Scylla Monitoring Stack
```

Reply:
I assume this is indeed the scope. Nothing in the page explain what is the goal here.

Reply:
I'll make it clearer, there are multiple reasons to use Thanos, this guide explains how to use Thanos as a data source instead of prometheus